views/NewChart.react.js

```diff
@@ -1,9 +1,69 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { compose } from 'redux'
+import { withStyles } from '@material-ui/core/styles'
+import { connect } from 'react-redux'
+
+import Header from './components/Header'
+import Sidebar from './components/Sidebar'
+
+import ChartForm from './forms/ChartForm'
+
+import getCounts from './fe-utils/countUtil'
+import getAvatar from './fe-utils/avatarUtil'
+import { fetchSubjects } from './fe-utils/fetchUtil'
+import getDefaultStyles from './fe-utils/styleUtil'
+import { chartStyles } from './styles/chart_styles'
+
+const NewChart = ({ user, classes }) => {
+  const [openDrawer, setOpenDrawer] = useState(false)
+  const [sideBarState, setSideBarState] = useState({ 
+    totalDays: 0,
+    totalStudies: 0,
+    totalSubjects: 0
+  })
+  const [avatar, setAvatar] = useState('')
+
+  const toggleDrawer = () => setOpenDrawer(!openDrawer)
+
+  useEffect(() => {
+    fetchSubjects().then(acl => {
+      setSideBarState(getCounts({ acl }))
+    })
+    setAvatar(getAvatar({ user }))
+  }, [])
 
-export default function NewChart() {
   return (
-    <div>
-      <h1> New Chart</h1>
+    <div className={classes.root}>
+      <Header
+        handleDrawerToggle={toggleDrawer}
+        title='Create a New Chart'
+        isAccountPage={false}
+      />
+      <Sidebar
+        avatar={avatar}
+        handleDrawerToggle={toggleDrawer}
+        mobileOpen={openDrawer}
+        totalDays={sideBarState.totalDays}
+        totalStudies={sideBarState.totalStudies}
+        totalSubjects={sideBarState.totalSubjects}
+        user={user}
+      />
+        <div className={`${classes.content} ${classes.contentPadded}`}>
+          <ChartForm 
```

Comment:
this `NewChart` component should still define the `onSubmit` handler and the `ChartForm` should call `props.onSubmit` with the form values on form submission.

Reply:
[78b1245](https://github.com/AMP-SCZ/dpdash/pull/84/commits/78b12458361f85aeb0919c885be4ca8c8f6c047b)